src/aiokatcp/sensor.py

```diff
@@ -186,6 +188,37 @@ def __init__(
         self.auto_strategy_parameters = tuple(auto_strategy_parameters)
         # TODO: should validate the parameters against the strategy.
 
+    def _check_value_type(self, value: _T) -> _T:
+        """Verify incoming value is compatible with sensor type.
+
+        This also handles the special case of :class:`core.Timestamp` where
+        it is used with `float` type interchangeably.
```

Comment:
I should either update this to discuss all accommodations, or remove it entirely from the docstring.

Reply:
I think it's fine for now. When we update the user documentation we can write a clear spec for what is accepted, and then this comment can be updated to refer to it.